platform/README.md

```diff
@@ -98,42 +62,153 @@ chmod +x '[path-to]/cortex/cortex-js/dist/src/command.js'
 npm link
 ```
 
+
+## Quickstart
+To run and chat with a model in Cortex:
+```bash
+# Start the Cortex server
+cortex
+
+# Start a model
+cortex run [model_id]
+```
+## Model Library
```

Comment:
Quick check: will Model Library be in the `cortex.cpp` README? (we should have it there)

We should refer to this as the Built-in Model Library

Reply:
Yes